package/MDAnalysis/transformations/translate.py

```diff
@@ -60,8 +61,69 @@ def wrapped(ts):
             v = np.float32(vector)
             ts.positions += v
         else:
-            raise ValueError("{} vector in too short".format(vector))
+            raise ValueError("{} vector is too short".format(vector))
         
         return ts
     
     return wrapped
+
+def center_in_box(ag, center='geometry', box=None, pbc=None):
+    """
+    Translates the coordinates of a given :class:`~MDAnalysis.coordinates.base.Timestep`
+    instance so that the center of geometry/mass of the given :class:`~MDAnalysis.core.groups.AtomGroup`
+    is centered on the unit cell.
+    
+    Example
+    -------
+    
+    .. code-block:: python
+    
+        ag = u.residues[1].atoms
+        ts = MDAnalysis.transformations.center(ag,center='mass')(ts)
+    
+    Parameters
+    ----------
+    ag: AtomGroup
+        atom group to be centered on the unit cell.
+    center: str, optional
+        used to choose the method of centering on the given atom group. Can be 'geometry'
+        or 'mass'
+    box: array
+        Box dimensions, can be either orthogonal or triclinic information.
+        Cell dimensions must be in an identical to format to those returned
+        by :attr:`MDAnalysis.coordinates.base.Timestep.dimensions`,
+        ``[lx, ly, lz, alpha, beta, gamma]``. If ``None``, uses these
+        timestep dimensions.
+    pbc: bool or None, optional
+        If True, all the atoms from the given AtomGroup will be moved to the unit cell
+        before calculating the center of mass or geometry
+    
+    Returns
+    -------
+    :class:`~MDAnalysis.coordinates.base.Timestep` object
+    """
+    
+    pbc_arg = pbc
+    if isinstance(ag, AtomGroup):
```

Comment:
Rather than explicitly test for the type of the argument, it would be better to try running the method and raise the correct error if it fails. This allows somebody to give as argument anything that roughly behave like an atom group. The python philosophy in that matter is that if it looks like a duck and quacks like a duck, then it is a duck. 

```python
try:
    ag_center = ag.center_of_geometry(...)
except AttributeError:
    raise ValueError('{:r} is not an AtomGroup'.format(ag))
```